### PR TITLE
Added example for `DOMException` constructor.

### LINKED ISSUE #6279: Issue with "DOMException()": Example is needed, currently is "TBD"
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DOMException/DOMException

#### What information was incorrect, unhelpful, or incomplete?
This is a follow-on to #6243.  The Example section says only "TBD".

#### Specific section or headline?
Example.

#### What did you expect to see?
An example of how to use DOMException().

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/domexception/domexception`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DOMException/DOMException
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/domexception/domexception/index.html
* Last commit: https://github.com/mdn/content/commit/bf3d143038cc4cba0b5f30e6f81428cf2d1d7fc0
* Document last modified: 2021-06-12T21:25:39.000Z

</details>

### PR DESCRIPTION
**Pull request by FarazzShaikh**
> What was wrong/why is this fix needed? (quick summary only)

`DOMException` constructor's example said "TBD".

MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DOMException/DOMException#Example

> Issue number (if there is an associated issue)

Fixes #6279

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thank you, this is a nice example. I'm going to nitpick about the way this section is organized - it's fine as it is but it's always good for pages to have a consistent structure.

The "Example' H2 should always be "Examples" plural even when there is only one of them.

Then it's best to have the prose description "In this example, pressing the button..." before the code.

Then it's good to have subheadings for each code block and for the output, like:

```
<h3>HTML</h3>

// the HTML

<h3>JavaScript</h3>

// the JS

<h3>Result</h3>

// the macro call

```

It's also good to omit the "hide-codepen-jsfiddle" argument - the only effect of this these days AFAICT is to stop the example iframe from getting a border, and a border is helpful to show that the example is not just part of the page.

**Comment by FarazzShaikh** [comment]
Fair enough, I will change the structure the way you described

**Comment by FarazzShaikh** [comment]
Done! And it does indeed look better. Thanks! Will keep these tips in mind in the future.

File: files/en-us/web/api/domexception/domexception/index.html [hunk 1]
```diff
@@ -36,9 +36,33 @@ <h3 id="Return_value">Return value</h3>
   <dd>A newly created {{domxref("DOMException")}} object.</dd>
 </dl>
 
-<h2 id="Example">Example</h2>
+<h2 id="Example">Examples</h2>
```
Comment (wbamberg) [inline files/en-us/web/api/domexception/domexception/index.html:39]:
```suggestion
<h2 id="Examples">Examples</h2>
```

File: files/en-us/web/api/domexception/domexception/index.html [hunk 2]
```diff
@@ -36,9 +36,33 @@ <h3 id="Return_value">Return value</h3>
...
+        throw new DOMException("Custom DOM Exception");
+    } catch(error) {
+        alert(`Error: ${error.message}`);
+    }
+}
+</pre>
+
+<h3>Result</h3>
+
+<p>{{ EmbedLiveSample('Example', '100%', 60) }}</p>
```
Comment (wbamberg) [inline files/en-us/web/api/domexception/domexception/index.html:65]:
```suggestion
<p>{{ EmbedLiveSample('Examples', '100%', 60) }}</p>
```

**Review by wbamberg — APPROVED** [review]
Looks great, thank you @FarazzShaikh !

**Pull request merged**